camel/agents/__init__.py

```diff
@@ -13,11 +13,11 @@
 # =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
 from .base import BaseAgent
 from .chat_agent import ChatAgent, ChatAgentResponse
-from .task_agent import TaskPlannerAgent, TaskSpecifyAgent
```

Comment:
Why are imports reshuffled? Do you use sort differently from what is there in the pre commit hook?

Reply:
actually, using `make format` actually helps me find several circular import bug, will fix in this pr too